app/models/manageiq/providers/redhat/infra_manager/host.rb

```diff
@@ -17,4 +17,8 @@ def verify_credentials(auth_type = nil, options = {})
 
     true
   end
+
+  def supports_quick_stats?
```

Comment:
change this to

```ruby
supports :quick_stats do
  ext_management_system.supports_quick_stats?
end
```

this way it makes `quick_stats` supported on you class too.

Also, as you are adding a feature, please add this to the `host.rb` on the vmware side
```ruby
supports :quick_stats
```